app/src/main/java/cm/aptoide/pt/home/AdsBundleViewHolder.java

```diff
@@ -52,27 +52,27 @@ public AdsBundleViewHolder(View view, PublishSubject<HomeEvent> uiEventsListener
     appsList.setNestedScrollingEnabled(false);
   }
 
-  @Override public void setBundle(HomeBundle homeBundle, int position) {
+  @Override public void setBundle(HomeBundle homeBundle) {
     if (!(homeBundle instanceof AdBundle)) {
       throw new IllegalStateException(this.getClass()
           .getName() + " is getting non AdBundle instance!");
     }
     bundleTitle.setText(
         Translator.translate(homeBundle.getTitle(), itemView.getContext(), marketName));
 
-    appsInBundleAdapter.updateBundle(homeBundle, position);
+    appsInBundleAdapter.updateBundle(homeBundle, getAdapterPosition());
```

Comment:
getAdapterPosition() only really needs to be used for listeners, as inconsistencies only arise there. 

I'm not 100% sure but I think it's better to use the position from onBindViewHolder for this line in particular, because it might be NO_POSITION if it's doing layout calculations.

This is also why in the listeners, a NO_POSITION check is also recommended. 

Check this https://youtu.be/imsr8NrIAMs?t=2149